src/Maatwebsite/Excel/Excel.php

```diff
@@ -83,8 +83,7 @@ public function create($filename, $callback = null)
         $writer->setTitle($filename);
 
         // Do the callback
-        if ($callback instanceof Closure)
```

Comment:
this still needs to be an if-statement. But it can be changed to `is_callable()`, else it will fail if somebody doesn't passes nothing.